tests/test_price_parsing.py

```diff
@@ -1960,6 +1962,23 @@ def __eq__(self, other):
 ]
 
 
+PRICE_PARSING_DECIMAL_SEPARATOR_EXAMPLES = [
+    Example(None, '1250€ 600',
+            '€', '1250', 1250, "€"),
+    Example(None, '1250€ 60',
+            '€', '1250€60', 1250.60, "€"),
+    Example(None, '1250€600',
+            '€', '1250€600', 1250.600, "€"),
+]
+
+PRICE_PARSING_DECIMAL_SEPARATOR_EXAMPLES_XFAIL = [
```

Comment:
I don't think we need XFAIL at all, as I see the fails show either that it produces unexpected outcome with wrong input or it produces wrong outcome with valid input (corner cases).

Reply:
Thanks for a quick review, I removed this one.